src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -53,8 +55,22 @@ public DefaultTargetService(ClientConfig clientConfig, TelemetryService telemetr
     } else {
       this.targetHttpClient = targetHttpClient;
     }
+
+    Properties defaultProps = new Properties();
+    try {
+      InputStream in = getClass().getResourceAsStream("/gradle.properties");
+      BufferedReader reader = new BufferedReader(new InputStreamReader(in));
+      defaultProps.load(reader);
```

Comment:
`load` also accepts an inputstream, so let's simplify:
```suggestion
      defaultProps.load(getClass().getResourceAsStream("/gradle.properties"));
```